src/main/java/net/dries007/tfc/client/RenderHelpers.java

```diff
@@ -155,6 +164,11 @@ public static void renderTexturedCuboid(PoseStack poseStack, VertexConsumer buff
         renderTexturedCuboid(poseStack, buffer, sprite, packedLight, packedOverlay, minX, minY, minZ, maxX, maxY, maxZ, 16f * (maxX - minX), 16f * (maxY - minY), 16f * (maxZ - minZ), doShade);
     }
 
+    public static void renderTexturedCuboid(PoseStack poseStack, VertexConsumer buffer, TextureAtlasSprite sprite, int packedLight, int packedOverlay, float minX, float minY, float minZ, float maxX, float maxY, float maxZ, int color)
```

Comment:
I may be wrong, but i think these pass throughs are just to pass an `int` color to the draw method? In that case I would just decompose the color with the bitwise `&` operator which is something we do in several places. If there is another eason then don't mind this comment.